en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md

```diff
@@ -0,0 +1,60 @@
+# Update Choreo Connect
+
+WSO2 introduces [WSO2 Updates](https://updates.docs.wso2.com/en/latest/) , which is a command-line utility that allows you to get the latest updates that are available for a particular product release. These updates include the latest bug fixes and security fixes that are released by WSO2 after a particular product version is released. Therefore, you do not need to wait and upgrade to the next product release to get these bug fixes.
+
+You can easily upgrade Choreo Connect [Using WSO2 Updates 2.0](https://updates.docs.wso2.com/en/latest/updates/update-tool/). The docker images related to Choreo Connect updates are reside in [WSO2 private docker registry](https://docker.wso2.com/) and the image tags are having the following format for Choreo Connect components.
```

Comment:
We should include the following.

1st Para
1. Public images - https://hub.docker.com/u/wso2
2. If you have a WSO2 subscription you can use the latest docker images with updates. These updates include the latest bug fixes and security fixes that are released by WSO2 after a particular product version is released. https://docker.wso2.com/

Topic: Docker Image Tags
You can have the below table.

Topic: Update Portal

You can use https://updates-info.wso2.com/ with sub details to login into update portal. You will be able to see the information like General description, Implementation details, Impact, Bug fixes and specific Instructions. Read more info - https://updates.docs.wso2.com/en/latest/updates/updates-portal/

Reply:
fixed via https://github.com/wso2/docs-apim/pull/6096/commits/ff7abbd3615e65f8560ce14ffc15025f6c9325fc